packages/aft/lib/src/utils/start_flutter_process.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+/// Starts a flutter [Process]
+///
+/// Passes [args] directly to [Process.start]
+/// Passes [package] to the [Process.start] workingDirectory parameter,
+/// after navigating to the amplify-flutter packages directory and appending "/example"
+/// Uses [printStream] to determine whether the events should be added to [stdout].
+Future<String> startFlutterProcess(
+  List<String> args,
+  String package, {
+  bool printStream = true,
+}) async {
+  stderr.write('Running "flutter ${args.join(' ')}".\n');
+  final output = <int>[];
+  final completer = Completer<void>();
+  final process = await Process.start(
+    'flutter',
+    args,
+    workingDirectory: '../../$package/example',
+  );
+  process.stdout.listen(
+    (List<int> event) {
+      output.addAll(event);
+      if (printStream) {
+        stdout.add(event);
+      }
+    },
+    onDone: () async {
+      return completer.complete();
+    },
+  );
+
+  await completer.future;
+  return Future<String>.value(utf8.decoder.convert(output).trim());
```

Comment:
```suggestion
  final process = await Process.run(
    'flutter',
    args,
    workingDirectory: '', // TODO
    stdoutEncoding: utf8,
    stderrEncoding: utf8,
  );
  if (printStream) {
    stdout.write(process.stdout);
    stderr.write(process.stderr);
  }
  if (process.exitCode != 0) {
    // TODO: This needs to be handled
  }
  return process.stdout as String;
```

Reply:
Using `Process.start` as opposed to `Process.run` enables us to print output to the console [as it is received](https://api.dart.dev/stable/2.17.3/dart-io/Process-class.html#:~:text=The%20run%20method%20executes%20the,interact%20with%20the%20running%20process.), which mimics the dx of running tests without this tool. I believe that that `Process.run` will send a result at the end... which could be a very long time depending on the tests being run.